Sources/TuistLoader/Models+ManifestMappers/TargetAction+ManifestMapper.swift

```diff
@@ -71,9 +71,17 @@ extension XcodeGraph.TargetScript {
             }
             let absolutePath = try generatorPaths.resolve(path: path)
             let base = try AbsolutePath(validating: absolutePath.dirname)
+            try processFile(at: absolutePath)
             return try base.throwingGlob(absolutePath.basename)
         }.reduce([], +)
     }
+
+    private static func processFile(at outputPath: AbsolutePath) throws {
+        let fileHandler = FileHandler.shared
+        if !fileHandler.exists(outputPath) {
+            try fileHandler.touch(outputPath)
+        }
```

Comment:
I'm not sure if this is the right way to go about it. Tuist should be able to generate references to files that don't exist without faking their existence.

Reply:
Ok! I'll try to do that